client/me/purchases/purchase-item/index.jsx

```diff
@@ -77,20 +72,26 @@ class PurchaseItem extends Component {
 			} );
 		}
 
-		if ( isDisconnectedSite ) {
-			if ( isJetpackTemporarySite ) {
+		if ( isDisconnectedSite && ! isAkismetTemporarySitePurchase( purchase ) ) {
+			if ( isJetpackTemporarySitePurchase( purchase ) ) {
 				return (
 					<>
 						<span className="purchase-item__is-error">
-							{ translate( 'Activate your product license key' ) }
+							{ translate( 'Activate your Jetpack product license key', {
+								components: {
+									br: <br />,
+									// TODO: These anchor links are causing React console warnings,
+									// "Warning: validateDOMNesting(...): <a> cannot appear as a descendant of <a>."
+									// Because the <CompactCard> component that renders this also us surrounded by an anchor link.
+									// See: <Card> General Guidelines: https://github.com/Automattic/wp-calypso/tree/trunk/packages/components/src/card#general-guidelines
+									// TLDR: Don't display more than one primary button or action in a single card. (in which the card itself if a primary action/link in this case)
```

Comment:
If I understand this correctly, you are removing the "Learn More" link for now until we can come back and fix this later correct?

If so, I know there are React warnings, but I feel as though we may want to keep this link here for now and leave this TODO comment so we can fix the warnings in the future. I think removing the link and leaving a TODO comment could leave it in limbo for a while, as I don't know when or who will get to this in the near future? Let me know what you think

Reply:
Oh my apologies, I probably should have added more context here by including a GH comment here with some additional explanation when I first posted this PR. 😉 .

No, the link is still there and showing. It looks the same, and is pointing to the same location. It was throwing a console warning before this PR just as it is still doing now. The only difference here is that originally the "Learn more text" was not being translated and it was being rendered by a separate component `<ExternalLink>{ children}</ExternalLink>`. So initially I thought maybe including the "Learn more" link in the same function call to`transate(..., {...} )` might fix the console warning, but it did not.  But still, it is translated now, so I just left it like that.

I could however put it back to the way it was, using `<ExternalLink>`, but make sure to `translate()` the "Learn more" text.. 
In fact, looking closer now, I think I will do that, because still, the way I have it now, I forgot to add `target="_blank"` and `rel="noopener noreferrer"`. Additionally, ExternalLink also adds some type of accessibility advantages for screen readers, I believe, too. Thanks for catching this, I'll get it fixed up. 👍

Oh, and as for the console warning, yes, the warning was already there before, and the fix, I feel is out of scope of this PR and would be best resolved in it's own separate PR.